assets/js/modules/sign-in-with-google/components/setup/SetupForm.js

```diff
@@ -61,9 +61,17 @@ export default function SetupForm() {
 				/>
 				<p className="googlesitekit-setup-module__step-description">
 					{ createInterpolateElement(
-						__(
-							'To set up Sign in with Google, Site Kit will help you create an "OAuth Client ID" that will be used to enable Sign in with Google on your website. You will be directed to a page that will allow you to generate an "OAuth Client ID". <a>Learn more</a>',
-							'google-site-kit'
+						sprintf(
+							/* translators: %1$s: Sign in with Google service name */
+							__(
+								'To set up %1$s, Site Kit will help you create an "OAuth Client ID" that will be used to enable %1$s on your website. You will be directed to a page that will allow you to generate an "OAuth Client ID". <a>Learn more</a>',
+								'google-site-kit'
+							),
```

Comment:
Shouldn't we use just `%s` instead of `%1$s`? We have just one placeholder anyway.

Reply:
That's because you have it in the `translators` comment. Replace it in the text to be `%s` and in the translators comment, then it should be good to go.

Reply:
Unfortunately that doesn't work:

![Screenshot 2024-12-10 at 12 44 09](https://github.com/user-attachments/assets/8bf1cdce-bf20-4983-8242-a8f404d929bc)

Oh, I get why: it's because I have multiple substitutions but only one positional argument. Rather than listing the `_x('Sign in with Google')` twice (as the second and third arguments) I can use `%1$s` to only use it once, which is easier to read.